resources/profiles/BBL/process/0.48mm Standard @BBL X1C 0.8 nozzle.json

```diff
@@ -8,7 +8,7 @@
     "compatible_printers": [
         "Bambu Lab X1 Carbon 0.8 nozzle",
         "Bambu Lab X1 0.8 nozzle",
-        "Bambu Lab P1S 0.6 nozzle",
+        "Bambu Lab P1S 0.8 nozzle",
```

Comment:
@SufficentMuffin 
I pushed a change here.
I think this is the better communication way :)
Now you won't see 0.48 for P1S 0.6 nozzle.
And for P1S 0.8 nozzle, it's available.